x/bank/keeper/keeper_test.go

```diff
@@ -28,15 +29,19 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
```

Comment:
```suggestion
	cmtbytes "github.com/cometbft/cometbft/libs/bytes"
```

Could you move this up too?